Resources/Locale/en-US/paper/paper-component.ftl

```diff
@@ -3,4 +3,8 @@
 
 # Shown when paper with words examined details
 paper-component-examine-detail-has-words = The paper has something written on it.
+# Shown when paper with stamps examined
+paper-component-examine-detail-stamped-by = It is stamped by: {$stamps}.
```

Comment:
this can probably be phrased better, maybe something like

```suggestion
paper-component-examine-detail-stamped-by = {CAPITALIZE(THE($paper)} {CONJUGATE-HAVE($paper)} been stamped by: {$stamps}.
```